### PR TITLE
Remove API endpoints for blockchain events

### PR DESCRIPTION
**Pull request by palango**
## Description

Removes `/api/_debug/blockchain_events/*`. Those used a full range RPC request which would most likely not work (on mainnet) and aren't actively used.

### CODE REVIEW

File: raiden/tests/integration/api/rest/test_rest.py [hunk 1]
```diff
@@ -557,138 +557,6 @@ def test_connect_insufficient_reserve(api_server_test_instance: APIServer, token
...
-    response = request.send().response
-    assert_proper_response(response, status_code=HTTPStatus.OK)
-    assert len(get_json_response(response)) > 0
-
-
-@raise_on_failure
-@pytest.mark.parametrize("number_of_nodes", [1])
-@pytest.mark.parametrize("channels_per_node", [0])
-@pytest.mark.parametrize("enable_rest_api", [True])
-def test_channel_events(api_server_test_instance: APIServer, token_addresses):
```
Comment (Dominik1999) [inline raiden/tests/integration/api/rest/test_rest.py:631]: why is this removed?

Comment (karlb) [reply raiden/tests/integration/api/rest/test_rest.py:631]: As @palango said:

> Removes /api/_debug/blockchain_events/*. Those used a full range RPC request which would most likely not work (on mainnet) and aren't actively used.

and this is the test for that part of the code, which should be removed along with the code itself.
